music21/test/testArticulations.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+import music21
+
+
+class TestArticulations(unittest.TestCase):  # pragma: no cover
+    def testArticulations(self):
+        score = music21.converter.parse('tab_test.xml')
+        guitar_part = score.parts[0]
+        notes = guitar_part.recurse().notes
+        self.assertEqual(int(notes[0].articulations[1].number), 7)
```

Comment:
the test should also check that the articulation matches the right class (is it a string or a fret?).  The int will not be necessary if the changes requested above are implemented,